docs/workspace.dsl

```diff
@@ -240,7 +240,7 @@ workspace "${SERVICE_NAME}" "Mozilla's service providing email and phone masks."
                     -> ga "Forwards extension's UI events" "Measurement Protocol"
                     -> email_sender "Sends email" "SES API"
                     -> sentry "Sends exceptions" "HTTPS API"
-                    -> metrics "Sends metrics" "HTTP"
+                    -> metrics "Sends metrics" "UDP"
```

Comment:
comment (non-blocking): same comment here: to understand the Relay architecture, does a Relay ENGR need to know that metrics are sent via UDP instead of HTTP?